src/main/java/io/fabric8/elasticsearch/plugin/OpenShiftElasticSearchService.java

```diff
@@ -65,31 +66,41 @@ protected OpenShiftElasticSearchService(final Settings settings, final Client es
 		this.cache = cache;
 		this.aclNotifier = aclNotifer;
 		restController.registerFilter(aclFilter);
-		restController.registerFilter(new KibanaUserReindexFilter(settings, logger));
+		
+		boolean reindexEnabled = settings.getAsBoolean(OPENSHIFT_KIBANA_REWRITE_ENABLED_FLAG, OPENSHIFT_KIBANA_REWRITE_ENABLED_DEFAULT);
+		
+		if ( reindexEnabled ) {
+			restController.registerFilter(new KibanaUserReindexFilter(settings, logger));
+		}
 	}
 
 	@Override
 	protected void doStart() throws ElasticsearchException {
+		
+		boolean dynamicEnabled = settings.getAsBoolean(OPENSHIFT_DYNAMIC_ENABLED_FLAG, OPENSHIFT_DYNAMIC_ENABLED_DEFAULT);
+
```

Comment:
Are you sure we need the notifier thread?  Originally, this was provided as a way to provide inter-thread communication where one was blocked awaiting for a 'callback' from the other.  I would think we would not need it if we are not rewriting the ACL